tests/CMakeLists.txt

```diff
@@ -4,8 +4,6 @@ enable_testing()
 find_package(GoogleTest REQUIRED)
 include(GoogleTest)
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
```

Comment:
Agree with this in general, but again, was added because someone had strong opinions on build directory layout to make running from build easier/possible.

Reply:
For reference, this change was what solved the 40ish commit hell of builds failing on MacOS. Unless someone can figure out another way to fix that, I'd rather not touch this particular change again.

Reply:
Was it related to mariadb linking / rpath?

Reply:
Yes